xclim/indices/_agro.py

```diff
@@ -113,3 +96,80 @@ def corn_heat_units(
 
     chu.attrs["units"] = ""
     return chu
+
+
+@declare_units(
+    tasmin="[temperature]",
+    tasmax="[temperature]",
+    thresh_tasmin="[temperature]",
+    thresh_tasmax="[temperature]",
+)
+def biologically_effective_degree_days(
+    tasmin: xarray.DataArray,
+    tasmax: xarray.DataArray,
+    thresh_tasmin: str = "10 degC",
+    thresh_tasmax: str = "19 degC",
+    lat_dim: str = "lat",
+    hemisphere: str = "north",
+) -> xarray.DataArray:
+    """
+
+    Parameters
+    ----------
+    tasmin: str
+      Minimum daily temperature.
+    tasmax: str
+      Maximum daily temperature.
+    thresh_tasmin: str
+      The minimum temperature threshold.
+    thresh_tasmax: str
+      The maximum temperature threshold.
+    lat_dim: str
+    hemisphere: {"north", "south"}
+      The hemisphere-based growing season to consider (north = April - October, south = October - April).
+
+    Returns
+    -------
+    xarray.DataArray
+    """
+    tasmin = convert_units_to(tasmin, "degC")
+    tasmax = convert_units_to(tasmax, "degC")
+    thresh_tasmin = convert_units_to(thresh_tasmin, "degC")
+    thresh_tasmax = convert_units_to(thresh_tasmax, "degC")
+
+    mask_tasmin = tasmin > thresh_tasmin
+
+    lat_mask = (abs(tasmin[lat_dim]) >= 40) & (abs(tasmin[lat_dim]) <= 50)
+    lat_constant = xarray.where(lat_mask, (abs(tasmin[lat_dim]) / 50) * 0.06, 0)
+
+    def sel_months(time):
```

Comment:
Not sure about this part... with xclim we typically want to give the users a choice via a `resample` frequency.  We could maybe see if we can somehow use the new  `aggregate_between_dates` that Marie Pier implemented? .